src/main/resources/db/migration/h2/V1.13__add-force-org-reset-flag.sql~HEAD

```diff
@@ -0,0 +1 @@
+ALTER TABLE `civil_servant` ADD `force_org_reset` bit(1) DEFAULT FALSE;
```

Comment:
Think this is a redundant file from the rebasing

Reply:
I spotted this when I was checking the code the other week ago.  Already deleted the duplicate file.